README.md

```diff
@@ -70,6 +70,29 @@ We recommend getting familiar with how [async/await](https://developer.mozilla.o
 npm i hashconnect --save
 ```
 
+#### Additional Steps to run Server-side
+
+> To enable running hashconnect running in a console app instead of a webapp, WebSocket and localStorage polyfills must be provided.
+
+1. Install polyfills
+```
+npm i websocket-polyfill --save
+npm i localstorage-polyfill --save
+```
+1. Import polyfills before using hashconnect
+```js
+import "websocket-polyfill";
+import 'localstorage-polyfill';
```

Comment:
Great work on tackling this. I think this could open up HashConnect to many more environments like game engines, Discord etc.

Although I'm sure these polyfills work great for getting a demo of this working in Node, I'd be careful using these in production since they're pretty stale packages and don't look to be maintained at all.

You might be able to swap `localstorage-polyfill` out with [node-localstorage](https://www.npmjs.com/package/node-localstorage) for example as it looks like it more recently maintained and has a lot more weekly downloads and stars on GitHub.

The other option of course is to have a Node specific implementation of HashConnect that uses [ws](https://github.com/websockets/ws) directly and also implements a simple in memory store to replace the localstorage. Then you wouldn't need to rely on the polyfills.

Reply:
Neither polyfills are required anymore. isomorphic-ws is used for the websocket connection and interactions with localstorage are no-ops if localstorage doesn't exist. In the future, abstracting this storage layer to a "bring your own storage" solution may be useful.